### DISCUSSION TITLE
美团，携程等app在付款时调用微信、支付宝支付无法正常调用

### DISCUSSION THREAD

**Discussion by cat-sir** [LSPosed/LSPatch#124] [Q&A]
大佬你好，情况如下：
微信、支付宝已经修补，也能正常使用，但美团和携程等App没有修补，在支付调用微信、支付宝提示未安装或支付宝版本过低，这类问题有解决方案或者考虑解决吗。
谢谢大佬

**Comment by myxuguiyuan** [comment]
把那些应用一起修补

**Comment by cat-sir** [comment]
好的，不过我root了，以后有机会再试试。谢谢大佬
